platform/broadcom/saibcm-modules/systems/linux/kernel/modules/knet-cb/psample-cb.c

```diff
@@ -509,7 +509,7 @@ psample_filter_cb(uint8_t * pkt, int size, int dev_no, void *pkt_meta,
         /* setup skb to point to pkt */
         memcpy(skb->data, pkt, meta.trunc_size);
         skb_put(skb, meta.trunc_size);
-        skb->len = meta.trunc_size;
+        skb->len = size;
```

Comment:
what about line 510 and 511, they are both meta.trunc_size

Reply:
Yes, that's the actual data we copy - which is the truncated size. If I were to change those to `size` we would over-read when the packet is truncated - as I understand the code.

Reply:
BRCM has agreed and fixed this as part of their next SAI release (7.1).
We shall be incorporating that change to roll it into the master branch soon
https://github.com/Azure/sonic-buildimage/pull/10583

Reply:
Thanks @gechiang, that's great news! Do you happen to know if this fix will be backported to 202011 as well?

Reply:
Unfortunately, BRCM is not backporting to earlier releases...